README.md

```diff
@@ -1,43 +1,420 @@
+    - [Overview](#overview)
+    - [Working with Image Relationships](#working-with-image-relationships)
+    - [Working with Direct Attachments](#working-with-direct-attachments)
+    - [Image Cropping and Sizes](#image-cropping-and-sizes)
+    - [Working with Focal Points](#working-with-focal-points)
+    - [Core Functions Reference](#core-functions-reference)
+  - [🚀 Deployment Options](#-deployment-options)
+    - [**ApostropheCMS Hosting** (Recommended)](#apostrophecms-hosting-recommended)
+    - [**DIY Deployment**](#diy-deployment)
+      - [Backend (ApostropheCMS) Deployment](#backend-apostrophecms-deployment)
+      - [Frontend (Astro) Deployment](#frontend-astro-deployment)
+      - [Netlify Deployment Example](#netlify-deployment-example)
+  - [📚 Learn More](#-learn-more)
+  - [🎯 Ready to Build Something Amazing?](#-ready-to-build-something-amazing)
 
-The provided `postinstall` script takes care of recursively running `npm install` for both `apostrophe` and `astro` when you run it at the top level. This is handy in both development and production.
 
-## `npm run update` in one step
+## 🎯 What This Starter Provides
 
-For convenience you can also run `npm run update` (NOTE: you must type the `run` part as npm has no `postupdate` hook for this). Just `npm update` won't work at the top level.
+This is a **minimal, non-opinionated foundation** that demonstrates the ApostropheCMS + Astro integration without imposing design decisions on your project. You get the essential building blocks:
 
-## Building for production
+- **Basic Page Types**: Home page, default content page, and simple blog structure
+- **Core Widgets**: Rich text, images, video, and a two-column layout widget
+- **Clean Architecture**: Well-organized but unopinionated code structure
+- **Integration Examples**: Working demonstrations of content fetching and rendering
 
-The provided `build` script covers production builds for both Astro and Apostrophe. You shouldn't need this in a development environment (see below).
+**Perfect for:** Developers who want to understand the integration and build their own design system on top, rather than those looking for a ready-to-launch theme.
 
-## Running in production
+## ✨ Why Use This Combination
 
-Note the `production-frontend-start` and `production-backend-start` scripts in `package.json`, which are executed by our hosting service to start Astro and Apostrophe, respectively. The presence of these commands is required in order for our hosting service to detect the presence of a combined project.
+**For Developers:**
+- 🏎️ **Blazing Fast**: Astro's server-side rendering + smart hydration = incredible Core Web Vitals scores
+- 🛠️ **Modern DX**: Write components in React, Vue, Svelte, or vanilla JS—your choice
+- 🔧 **Zero API Boilerplate**: The `apostrophe-astro` package handles all the backend communication automatically
+- 📡 **Flexible Deployment**: SSR hosting on Netlify, Vercel, Cloudflare, or any Node.js environment
```

Comment:
Why aren't we promoting our hosting service for easy combined apostrophe + astro hosting? I'm not against mentioning others of course, lock-in is unsettling